css/sidebar.css

```diff
@@ -0,0 +1,69 @@
+.sidebar{
+    grid-area: sidebar;
+    position: sticky;
+    top: 0rem;
+    height: 100vh;
+    overflow-y: auto;
+    max-width: 225px;
```

Comment:
do you think it's better to mention the width as part of `grid-template-area` only?
Ex: instead of giving `1fr` - you can have `grid-template-columns: 225px 1fr 1fr 1fr 1fr 1fr;`

Reply:
That would give the `sidebar` a fixed width then.  For smaller windows, I don't want the sidebar to occupy more space than the components. 
Also, I just noticed that I don't need `max-width`. Will remove it